jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/SystemConfigMBean.java

```diff
@@ -0,0 +1,17 @@
+package us.dot.its.jpo.ode;

```

Comment:
The interface and systemconfig class are necessary for the jconsole application to be able to connect to the ODE